pennylane/gradients/general_shift_rules.py

```diff
@@ -366,11 +366,12 @@ def generate_multi_shift_rule(frequencies, shifts=None, orders=None):
 
     This corresponds to the gradient rule
 
+    TODO THERE IS A BUG CHECK RENDERING
     .. math::
 
         \frac{\partial^2 f}{\partial x\partial y} &= \frac{1}{4}
         \left[f(x+\pi/2, y+\pi/2) - f(x+\pi/2, y-\pi/2)\\
-        &~~~- f(x-\pi/2, y+\pi/2) + f(x-\pi/2, y-\pi/2) \right].
+        &- f(x-\pi/2, y+\pi/2) + f(x-\pi/2, y-\pi/2) \right].

```

Comment:
You should be able to use a latex align environment in ReST math:

```
.. math::

    \begin{align*}
    \beta_k & = \frac{2i\gamma}{2n+1}\sum_{j=-n}^n j \omega_k^j
    \end{align*}
```

(there are plenty of demos doing that)

Reply:
Thanks! Yeah I'm a bit confused why this one does not render either way :thinking: Will figure it out!